bin/reth/src/chain/mod.rs

```diff
@@ -0,0 +1,53 @@
+use clap::Parser;
+use eyre::WrapErr;
+use reth_primitives::Block;
+use std::path::PathBuf;
+
+#[derive(Debug, Parser)]
+pub struct ImportCommand {
+    /// The file to import.
+    path: PathBuf,
+}
+
+impl ImportCommand {
+    pub async fn execute(&self) -> eyre::Result<()> {
+        // read file
+        let mut contents = &std::fs::read(&self.path).wrap_err("Could not open file")?[..];
+
+        // open or create db
+        let mut imported = 0;
+        while !contents.is_empty() {
+            match <Block as reth_rlp::Decodable>::decode(&mut contents) {
+                Ok(block) => {
+                    println!("{:?}", block.header.hash_slow());
+                    imported += 1;
+                }
+                Err(err) => {
+                    println!("Error: {:?}", err);
```

Comment:
Are we sure here? Don't we want to error?

Reply:
This is just super draft for me to show that I am working on it, nothing is final. This will most likely be an error but I am working off of how geth does it